src/test/java/marquez/db/DatasetDaoTest.java

```diff
@@ -61,13 +66,31 @@ public static void setUpOnce() {
     datasetDao = jdbi.onDemand(DatasetDao.class);
   }
 
+  @Before
+  public void setup() {
```

Comment:
Not every tests requires rows to be inserted. I think if that was the case, a setup method like this one would make a lot of sense. To avoid confusion and created test rows unnecessarily, I'd move these inserts into their respective tests.

Reply:
There are only two test cases that don't rely on `datasourceRow` and `namespaceRow`. I've modified one of them to use the `namespaceRow` created in setup (which it should have been doing anyway), so that leaves just one.
I'm okay with one test method not needing the setup code but having it run anyway. 

The alternative would be to re-add 22 lines of duplicate code, which would take away from the readability and break the DRY principle, since we're trying to accomplish the same setup each time.